README.md

```diff
@@ -163,6 +163,16 @@ you can follow the [documentation of limit count](doc/plugins/limit-count.md).
 
 Then you can try more [plugins](doc/README.md#plugins).
 
+## Deploy to the Cloud
+
+### AWS
+
+The recommended approach is to deploy APISIX with [AWS CDK](https://aws.amazon.com/cdk/) on [AWS Fargate](https://aws.amazon.com/tw/fargate/) which helps you decouple the APISIX layer and the upstream layer on top of a fully-managed and secure serverless container compute environment with autoscaling capabilities.
```

Comment:
I think `https://aws.amazon.com/fargate/` would be better

Reply:
Sure. Resolved.